### PR TITLE
Academic reminders

### PR DESCRIPTION
**Pull request by sailoraria**
Created Assignment and Exam entities to store scheduled academic events.
Created an EventDate entity which is used to store the date of the events.
Created a Schedule entity which was not implemented.

Updated the readme to add my academic reminders use case.

Changed the Main class back after testing my entities to match what is in the main branch.  

Also had to manually add back some deleted files so hopefully it works well!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,8 +40,16 @@ You can find all the user stories and other details in our blueprint from [here]
...
-  - [ ] for assignments
+  - [X] Create a scheduled assignment by providing the course code, due date, and name of the assignment. (July 5)
+  - [X] Create a scheduled exam by providing the course code, location, and date (day + time) of the exam. (July 5)
+  - [ ] Bot sends direct message reminders when prompted.
+  - [ ] Bot sends scheduled direct message reminders.
+  - [ ] Bot returns a schedule with outstanding academic events (incl. both assignments and exams).
+  - [ ] Bot accepts an exam schedule and creates reminders for each exam.
+
+- **Schedule** 📅 (use case in progress - July 12)
+  - [ ]
```
Comment (xinyue296) [inline README.md:51]: perhaps removing the blank line will be better?

**Review by xinyue296 — APPROVED** [review]
Looks good to me! Looking forward to the reminder feature!!!

**Review by Overpoweredplatypus — APPROVED** [review]
Approved! 🗣️

**Review by christianjokhaji — APPROVED** [review]
Looks good to me! Great work. 👌

**Pull request merged**